### PR TITLE
Added logos and social to press centre

### LINKED ISSUE #132: Build: insights /press-centre from copydoc
insights.ubuntu.com/press-centre/ - copy doc (vanilla insights): https://docs.google.com/document/d/1liiT3l86LHfIyYd_Neg5LAAMRa44VF5QlF63S2gph3Q/edit#

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Created a new press-centre layout
- Added logos and social to press centre
- Still, need to add an Announcement based archive (see #146)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [the press centre](http://0.0.0.0:8023/press-centre/)
- See that the list matches the [copy doc](https://docs.google.com/document/d/1liiT3l86LHfIyYd_Neg5LAAMRa44VF5QlF63S2gph3Q/edit#)

## Issue / Card

Fixes #132

### CODE REVIEW

File: templates/press-centre.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+          {% endif %}
+        </ul>
+      </div>
+    </nav>
+  </section>
+
+  <div class="p-strip is-bordered">
+    <div class="row">
+      <div class="col-8">
+
```
Comment (deadlight) [inline templates/press-centre.html]: Some weirdness here with indenting/nesting

**Review by deadlight — CHANGES_REQUESTED** [review]
Comment in code.

Also, there's no gap between the last row of items nd the footer.

Lastly, there's a conflict.

Other than those small things, all looks good

File: templates/press-centre.html (hunk 1 above)
Comment (pmahnke) [reply templates/press-centre.html]: fixed

**Comment by pmahnke** [comment]
@deadlight ready for another go

**Review by deadlight — APPROVED** [review]

**Comment by frankiegranato** [comment]
I can't see the demo, instead I see a page that says: IndexError: list index out of range :-(

**Comment by pmahnke** [comment]
@frankiegranato try this - http://insights.ubuntu.com-pr-147.run.demo.haus/press-centre/

**Comment by frankiegranato** [comment]
I think we could improve the design and content-strategy for this page but, for this stage in time, ehat we have is great.

**Pull request merged**